pkg/kube/inject/webhook.go

```diff
@@ -480,7 +480,15 @@ func escapeJSONPointerValue(in string) string {
 // adds labels to the target spec, will not overwrite label's value if it already exists
 func addLabels(target map[string]string, added map[string]string) []rfc6902PatchOperation {
 	patches := []rfc6902PatchOperation{}
-	for key, value := range added {
+
+	var addedKeys []string
+	for key := range added {
+		addedKeys = append(addedKeys, key)
+	}
```

Comment:
nit: unnecesserary memory reallocation for map key copying? wouldn't:
```
    addedKeys := make([]string, 0, len(added))
    for key := range added {
        addedKeys = append(addedKeys, key)
    }
```
be a bit more efficient?

Reply:
perhaps. these should be lists of 2 max. but the most efficient would be to not have to sort at all. unfortunately, that would require updating the golden file-based tests to have a canonical serialization before comparison instead of raw bytes. fortunately, the webhook isn't the most perf-sensitive component, so this is probably not noticeable.